test/functional/mempool_compatibility.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+# Copyright (c) 2017-2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test that mempool.dat is compatible between versions
+
+Download node binaries:
+contrib/devtools/previous_release.sh -b v0.19.1
+"""
+
+import os
+
+from test_framework.test_framework import BitcoinTestFramework
+
+class MempoolCompatibilityTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.setup_clean_chain = True
```

Comment:
If you remove this line, you can also remove the whole "Generate some coins on both wallets" part

Reply:
I didn't work for me for some reason, though I don't fully understand yet how chain generation works in tests. I'll try this again tomorrow and will get back with an update

Reply:
I figured it out. I forgot to manually import coinbase keys in setup_nodes(), it works now.
Thanks for the suggestion.